apps/publications/src/app/pages/create-publication-page/import-publications-page/import-publications-page.component.html

```diff
@@ -22,19 +22,19 @@ <h1 class="page-subtitle d-flex">
       <perun-web-apps-year-range [startYear]="startYear" [endYear]="endYear">
       </perun-web-apps-year-range>
 
-      <button mat-stroked-button class="left-space" (click)="searchPublications()">
+      <button mat-stroked-button class="start-space" (click)="searchPublications()">
```

Comment:
Again incorrect replacement. In this class is more similar cases like that for `start-space` and also for `end-button`.